autoinstall_script/README.md

```
@@ -0,0 +1,298 @@
+| Logs  | `journalctl -u ivorysql -f`| Follow service logs        |
+| Reload| `systemctl reload ivorysql`| Reload configurations      |
+| Connect | `sudo -u ivorysql /usr/ivorysql/bin/psql` | Connect to DB |
+| Version | `/usr/ivorysql/bin/postgres --version` | Show version |
+| Base Backup | `sudo -u ivorysql /usr/ivorysql/bin/pg_basebackup` | Create base backup |
+
+---
+
+## 4. Troubleshooting
+
+### 4.1 Common Error Handling
+| Symptom | Likely Cause | Resolution |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)